templates/project.html

```diff
@@ -21,29 +21,33 @@ <h1>{{ this.name }}</h1>
     {{ incomplete(this) }}
 
     {% if this.attic or "attic" in this.maturity %}
-    <p><i>{{ "attic_before"|trans }} <a href="{{ '/project/attic'|url(alt=this.alt) }}">{{ "attic_link"|trans }}</a>{{ "attic_after"|trans }}</i></p>
+    <p><i>{% trans attic_link='/project/attic'|url(alt=this.alt) %}
+      This project is classified as <a href="{{ attic_link }}">in the attic</a>. It is no longer actively maintained
```

Comment:
Should I fix missing period here?
```suggestion
      This project is classified as <a href="{{ attic_link }}">in the attic</a>. It is no longer actively maintained.
```